app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -11,12 +11,15 @@
 .gem-c-pagination__item {
   @include govuk-font($size: 16, $line-height: (20 / 16));
   list-style: none;
+  height: 70px;
+  border-top: 1px solid $govuk-border-colour;
+  margin-bottom: 16px;
 }
 
 .gem-c-pagination__link {
   @extend %govuk-link;
   display: block;
-  padding: govuk-spacing(3);
+  padding: 15px 0 20px 0;
```

Comment:
Do we not try and avoid these various explicit px numbers in favour of using common spacing from govuk-frontend?

Reply:
have applied the appropriate govuk-spacing values